src/linear_algebra/bilinear_map.lean

```diff
@@ -120,7 +124,7 @@ theorem map_sub₂ (f : M' →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂] P') (x y z)
 theorem map_add₂ (f : M →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂] P) (x₁ x₂ y) : f (x₁ + x₂) y = f x₁ y + f x₂ y :=
 (flip f y).map_add _ _
 
-theorem map_smul₂ (f : M →ₗ[R] N →ₗ[S] Pₗ) (r : R) (x y) : f (r • x) y = r • f x y :=
+theorem map_smul₂ (f : M₂ →ₗ[R] N₂ →ₛₗ[σ₁₂] P₂) (r : R) (x y) : f (r • x) y = r • f x y :=
```

Comment:
Is there a reason why this lemma is only semilinear in the second argument?

Reply:
if you have a map that is semilinear in the first argument, then you can use `map_smulₛₗ₂`. The change here is that you only need to be linear in the first argument and not in the second.for `f (r • x) y = r • f x y` to hold.

Reply:
if you have a map that is semilinear in the first argument, then you can use `map_smulₛₗ₂`. The change here is that you only need to be linear in the first argument and not in the second.for `f (r • x) y = r • f x y` to hold.

Reply:
Ah I see, yes that makes sense.